docs/package_integration/oscal-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Oscal
```

Comment:
```suggestion
# OSCAL
```
minor but for consistency's sake

Reply:
Unfortunately OSCAL is an acryonym

OSCAL: the Open Security Controls Assessment Language

I will update, to include the definition

Reply:
yeah tracking thats why i asked for it all caps! 😄

Reply:
yikes. that's embarassing _AND_ now it's on the internet. sec gonna go delete my github profile and recreate. ;)

Reply:
10/10 proportionate action, no notes 🫡